.github/workflows/skip-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Skip CI
+on:
+  workflow_call:
+    outputs:
+      skip_ci:
+        description: "Value yes if the current PR only contains changes to files not related to code, otherwise, not defined."
+        value: ${{ jobs.check-ci.outputs.skip_ci }}
+
+jobs:
+  check-ci:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      skip_ci:
+        ${{ steps.check_diff.outputs.skip_ci }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check diff from Pull Request
+        id: check_diff
+        run: |
+          skipList=(".github/CODEOWNERS" ".prettierignore")
+          # Ignores changelog.md, readme.md,...
+          fileChangesArray=($(git diff --name-only  origin/${{ github.base_ref }}..origin/${{ github.head_ref }} | grep -v '\.md$'))
```

Comment:
We only have 5 md files, could we just add them to the skip list to make this more explicit? 

![Screenshot 2024-01-31 at 12 19 00](https://github.com/getsentry/sentry-react-native/assets/31292499/3ea7e2dc-9536-4f66-8235-20354ebbdd2e)